src/Symfony/Component/Form/Extension/Core/Type/IntegerType.php

```diff
@@ -25,7 +25,13 @@ class IntegerType extends AbstractType
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder->addViewTransformer(new IntegerToLocalizedStringTransformer($options['grouping'], $options['rounding_mode']));
+        $builder->addViewTransformer(
+            new IntegerToLocalizedStringTransformer(
```

Comment:
please keep this on one line

Reply:
Sorry, PHPStorm transfers it to new line when reformat code. :( It's too long.

Reply:
What's the issue with multi lines anyway? It's according to the standarts. Now it's very hard to read because I have to scroll horizontally. :(